### DISCUSSION TITLE
gauge-js plugin installation from zip

### DISCUSSION THREAD

**Discussion by PeterKovacs1** [getgauge/gauge#2353] [General]
Trying to install gauge-js plugin from zip. It seems that that plugin.json is not within the zip archive.

```
gauge install js --file gauge-js-2.4.2.zip
Failed to install plugin 'js'.
Reason: File /tmp/gauge_temp1687496358582952221/plugin.json doesn't exist.
```
Can anyone help me to get gauge-js installed? Pulling over network is unfortunately restricted.

I am running gauge-1.4.3-linux.x86_64

The installation of gauge-js-2.4.0.zip worked fine.

**Comment by haroon-sheikh** [comment]
@PeterKovacs1 can you try the latest version 3.0.1 of gauge-js please and let us know if it fixes the issue for you.

**Reply by PeterKovacs1** [reply]
@haroon-sheikh Version 3.0.1 works - thanks!
